src/pages/NotFound/Components/Logos/Logos.tsx

```diff
@@ -0,0 +1,11 @@
+import './Logos.scss';
+import logos from './photos/image 1174.png';
+export const Logos =()=>{
+    return(
+        <div className='logos-container'>
+            <div className='logos-image'>
+                <img src={logos} alt='...' />
```

Comment:
in a real project you must have 5 images here))
but let's say that's ok for now)